superset-frontend/cypress-base/cypress/e2e/dashboard/nativeFilters.test.ts

```diff
@@ -225,11 +225,11 @@ function setFilterBarOrientation(orientation: 'vertical' | 'horizontal') {
   }
 }
 
-function openMoreFilters(intercetFilterState = true) {
+function openMoreFilters(waitFilterState = true) {
   interceptFilterState();
   cy.getBySel('dropdown-container-btn').click();
 
-  if (intercetFilterState) {
+  if (waitFilterState) {
```

Comment:
This typo meant that people didn't realize that they were almost shadowing a function. I've renamed the argument to `wait` instead of `intercept` to more accurately describe what the boolean guards...